docs/src/app/components/DatePicker/ExampleDatePickers.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import DatePicker from 'material-ui/lib/date-picker/date-picker';
+import TextField from 'material-ui/lib/text-field';
+import Toggle from 'material-ui/lib/toggle';
+
+
+const DatePickerExamples = React.createClass({
+  getInitialState() {
+    let minDate = new Date();
+    let maxDate = new Date();
+    minDate.setFullYear(minDate.getFullYear() - 1);
+    minDate.setHours(0, 0, 0, 0);
+    maxDate.setFullYear(maxDate.getFullYear() + 1);
+    maxDate.setHours(0, 0, 0, 0);
+
+    return {
+      minDate: minDate,
+      maxDate: maxDate,
+      autoOk: false,
+      controlledDate: new Date('2015/07/15'),
+    };
+  },
+
+  render() {
+    let optionsStyle = {
```

Comment:
No need  to allocate the memory for the style at each render, can we move it outside?